File: Java_Codes/6. Collections Framework/HashSetExample.java
package Collections;

import java.util.HashSet;

public class HashSetExample {
    public static void main(String[] args) {
        // Create a HashSet of strings
        HashSet<String> fruits = new HashSet<>();

        // Add elements to the HashSet
        fruits.add("Apple");
        fruits.add("Banana");
        fruits.add("Orange");
        fruits.add("Mango");
        fruits.add("Apple"); // Duplicate element

        // Print the HashSet
        System.out.println("HashSet: " + fruits);

        // Check if an element exists
        if (fruits.contains("Banana")) {
            System.out.println("Banana is in the HashSet.");
        }

        // Remove an element
        fruits.remove("Orange");
        System.out.println("HashSet after removal: " + fruits);
    }
}

File: Projects/PythonProject/Collections_Framework/Collections/HashSetExample.py
def main():
    # Create a set of strings
    fruits = set()

    # Add elements to the set
    fruits.add("Apple")
    fruits.add("Banana")
    fruits.add("Orange")
    fruits.add("Mango")
    fruits.add("Apple")  # Duplicate element, will be ignored

    # Print the set
    print("Set:", fruits)

    # Check if an element exists
    if "Banana" in fruits:
        print("Banana is in the set.")

    # Remove an element
    fruits.discard("Orange")
    print("Set after removal:", fruits)


if __name__ == "__main__":
    main()
